### PR TITLE
Adding a tips and tricks section to edge book, fixing a typo in edge.adoc

### PR DESCRIPTION
**Pull request by Gagrio**

### CODE REVIEW

**Comment by Gagrio** [comment]
@atanasdinov @ipetrov117 @jdob you guys feel such section would make sense to have here or is it better suited for upstream docs? thanks!

File: asciidoc/tips/eib.adoc [hunk 1]
```diff
@@ -0,0 +1,123 @@
+= *Edge Image Builder Tips and Tricks*
+
+_This is a Work In Progress_
```
Comment (e-minguez) [inline asciidoc/tips/eib.adoc:3]: If we are going to merge this, this needs to be removed.

File: asciidoc/edge-book/edge.adoc [hunk 2]
```diff
@@ -94,6 +94,16 @@ include::../guides/metallb-kube-api.adoc[leveloffset=+1]
 
 include::../guides/air-gapped-eib-deployments.adoc[leveloffset=+1]
 
+//--------------------------------------------
+// Tips and Tricks
+//--------------------------------------------
+
+= Tips and Tricks
+[partintro]
+Some useful tips and tricks
```
Comment (e-minguez) [inline asciidoc/edge-book/edge.adoc:103]: I think this is redundant (?)

File: asciidoc/tips/eib.adoc [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+.Common 
+- Podman machines are generally low on resources (especially on MacOS), therefore you might need to increase the machine's resources to run more competitive tasks (for example downloading RPMs).
```
Comment (e-minguez) [inline asciidoc/tips/eib.adoc:6]: I wouldn't say "generally low on resources" but "by default".
Also podman "machine" is only needed on non-linux OS (both OSX and Windows).

Comment (e-minguez) [reply asciidoc/tips/eib.adoc:6]: I would also mention how to do it :)

File: asciidoc/tips/eib.adoc [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+- Make sure you're building an image for the appropriate architecture, especially if you're not on a x86_64 environment, as the installation might complete, but the applications will not start, in the best case scenario.
```
Comment (e-minguez) [inline asciidoc/tips/eib.adoc:7]: I would rephrase this as "the image architecture being built must match the one on the system running the EIB container"  or something along those lines.

File: asciidoc/tips/eib.adoc [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+image:
+  arch: aarch64
+  imageType: iso
+  baseImage: slemicro.iso
+  outputImageName: single-node-k3s.iso
+----
+
+.Kubernetes
+- You can't create an image for a multi node kubernetes cluster without setting up a networking section in your definition file, more information https://github.com/suse-edge/edge-image-builder/blob/main/docs/building-images.md#kubernetes[here]
+- You can't create an image for a multi node kubernetes cluster without setting up an apiVIP (even better add an apiHost to the setup) in the network section of your definition file, more information https://github.com/suse-edge/edge-image-builder/blob/main/docs/building-images.md#kubernetes[here]
```
Comment (e-minguez) [inline asciidoc/tips/eib.adoc:19]:
```suggestion
- You can't create an image for a multi node kubernetes cluster without setting up an `apiVIP` (even better add an `apiHost` to the setup) in the network section of your definition file, more information https://github.com/suse-edge/edge-image-builder/blob/main/docs/building-images.md#kubernetes[here]
```

File: asciidoc/tips/eib.adoc (hunk 3 above)
Comment (Gagrio) [reply asciidoc/tips/eib.adoc:6]: on it

File: asciidoc/tips/eib.adoc (hunk 4 above)
Comment (Gagrio) [reply asciidoc/tips/eib.adoc:7]: changing it, thanks

File: asciidoc/tips/eib.adoc [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+  - destination: 0.0.0.0/0
+    metric: 100
+    next-hop-address: 192.168.64.1
+    next-hop-interface: eth0
+    table-id: 254
+  - destination: 192.168.100.0/24
+    metric: 100
+    next-hop-address:
+    next-hop-interface: eth0
+    table-id: 254
```
[the remark this answers is not in the document]
Comment (Gagrio) [reply asciidoc/tips/eib.adoc:67]: ok

**Comment by Gagrio** [comment]
Closing the PR to avoid rebasing and conflict resolution since it's beed open quite a while. 
Will address the comments above and create a new PR.
